### PR TITLE
Create HelloWorld.html

### PR DESCRIPTION
**Pull request by shirley92**
Hey check it.

### CODE REVIEW

File: HelloWorld.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en-US">
+<body>
+
+<h1>Jobs in Canada</h1>
+Find public and private sector job opportunities and hiring programs, apply or extend a work permit, get a Social Insurance Number, a criminal record check or a security clearance..
+<p>Categories</p>
+<ul>
+  <li><a href="https://itsgodsbelovedchild.tumblr.com">HR</a></li>
+  <li>IT</li>
+  <li>Finance</li>
```
Comment (imvijay2007) [inline HelloWorld.html:11]: Hey, I think you forgot another sector - Human Resources. Can you please add it to the list? Thanks.

Comment (shirley92) [reply HelloWorld.html:11]: Okay

File: HelloWorld.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+</ul>
+<button>Apply</button>
```
Comment (imvijay2007) [inline HelloWorld.html]: Also please change the text from "Apply" to "Apply Now!"

**Review by imvijay2007 — CHANGES_REQUESTED** [review]
Can you make changes so that I can merge?

**Review by imvijay2007 — APPROVED** [review]

**Comment by imvijay2007** [comment]
Changes look good. Thank you @shirley92 ! Merging to master now.

**Pull request merged**